testing-mysql-server-base/src/main/java/com/facebook/presto/testing/mysql/AbstractEmbeddedMySql.java

```diff
@@ -110,6 +113,16 @@ public Connection getMySqlDatabase()
         return DriverManager.getConnection(getJdbcUrl("root", "mysql"));
     }
 
+    protected String getMysqlInstallDb()
+    {
+        return (isMariadb ? serverDirectory.resolve("bin").resolve("mysql_install_db").toString() : "");
```

Comment:
if this doesn't apply to non-mariadb, make the method name getMariaDBInstallDB and have it throw a verify exception if isMariaDb is false.